test/ci/kokoro/run_integ_tests.sh

```diff
@@ -68,10 +68,6 @@ function install_pyenv {
     export PATH="$PYENV_ROOT/bin:$PATH"
     eval "$(pyenv init --path)"
   fi
-  pyenv update
```

Comment:
Maybe leave a comment saying that if we start seeing kokoro failing because of a certain python version missing, we can do `pyenv update`.  That way we don't have to spend time investigating why the version is not found.

IIRC, the update was added at that time because python 3.8 was missing. Looks like the new pool probably ships with pyenv that is aware of 3.8, but it might happen that certain future Python versions are not available with the available pyenv.

Reply:
`pyenv update` just isn't a command unless we also want to install the `pyenv-update` package

https://github.com/pyenv/pyenv#upgrading-with-installer-or-git-checkout

https://github.com/pyenv/pyenv#upgrading-with-installer-or-git-checkout